File: backend/java/src/main/java/com/georgeherby/backend/models/ItemsResponse.java
package com.georgeherby.backend.models;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.ArrayList;
import java.util.List;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.NonNull;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class ItemsResponse {

  @JsonProperty("items")
  @NonNull
  private ArrayList<Product> productList;

}

File: backend/csharpdotnet-3/dotnetcore3/Models/ItemsResponse.cs
using System;
using System.Collections.Generic;

namespace dotnetcore3.Models
{
    public class ItemsResponse
    {
        public List<Product> ProductList { get; set; }

        public ItemsResponse(List<Product> productList)
        {
            ProductList = productList;
        }
    }
}

